src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformer/ExceptionRecordToCaseTransformer.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformer;
+
+import org.mapstruct.Mapping;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.in.ExceptionRecord;
+
+import java.util.Map;
+
+public interface ExceptionRecordToCaseTransformer {
+    Map<String, Object> toCase(ExceptionRecord exceptionRecord);
```

Comment:
Maybe a more descriptive name?
Please have a look into the Divorce review: https://github.com/hmcts/div-case-orchestration-service/pull/596

Reply:
transformIntoCaseData as on Divorce